portfolio/src/main/webapp/style.css

```diff
@@ -1,9 +1,376 @@
-#content {
-  margin-left: auto;
-  margin-right: auto;
-  width: 650px;
+@import url('https://fonts.googleapis.com/css2?family=Comfortaa:wght@300;400&display=swap');
+* {
+  box-sizing: border-box;
 }
 
-#greeting-container {
+body, html {
+  background-color: #474e5d;
+  height: 100%;
+  margin: 0;
+  font-family: 'Comfortaa', cursive;
+}
+
+/*Home View with Background*/
+header{
+  height: 100%;
+  background: linear-gradient(rgba(12, 12, 12, 0.99),rgba(12, 12, 12, 0.3)), 
+      url("/images/sunset.jpg") no-repeat fixed center;
+  color: #fcfafc;
+  background-size: cover;
+}
+
+/*Sections*/
+section {
+  height: auto;
+  padding:100px 100px;
+}
+
+.dark {
+  background-color:#0C0C0C;
+  color: #fcfafc;
+}
+
+.light {
+  background-color: #fcfafc;
+  color: #0C0C0C;
+}
+
+.blue {
+  background-color: #c7e8ff;
+  color: #0C0C0C;
+}
+
+/*Containers*/
+.container {
+  position: relative;
+  margin: 50px 50px;
+  text-align:center
+}
+
+/*Text Wrappers*/
+.abs-center {
+  margin: 0;
+  position: absolute;
+  top: 50%;
+  left: 50%;
+  text-align: center;
+  -ms-transform: translate(-45%, -50%);
```

Comment:
jw, why only the ms prefix?

Reply:
oh I just realized what the prefixes mean. I probably took this snippet from how to center things from somewhere. that line isn't needed. I deleted. haha ty for catching that